src/main/java/duke/task/ToDoTask.java

```diff
@@ -0,0 +1,19 @@
+package duke.task;
+
+/**
+ * Inherit from Task.
+ * This is a ToDoTask.
+ */
+public class ToDoTask extends Task {
+    public ToDoTask(String taskName) {
+        super(taskName);
+    }
+    public ToDoTask(String taskName, boolean isDone) {
```

Comment:
Consider adding a line separation between the two constructors.

Reply:
Noted. Thanks for pointing it out!